src/Symfony/Component/Workflow/Registry.php

```diff
@@ -43,7 +43,10 @@ public function has(object $subject, string $workflowName = null): bool
         return false;
     }
 
-    public function get(object $subject, string $workflowName = null): Workflow
+    /**
+     * @return WorkflowInterface
+     */
+    public function get(object $subject, string $workflowName = null)
```

Comment:
If there was already a native return type, why widening it to `mixed` to restrict it again in the next major version instead of widening it to WorkflowInterface ?